telemetry/definitions/commonDefinitions.json

```diff
@@ -850,6 +905,86 @@
             "name": "apprunner_viewServiceLogs",
             "description": "View the App Runner service logs (the logs produced by App Runner)"
         },
+        {
+            "name": "auth_gettingStartedPage",
+            "description": "Called when user lands on getting started page",
+            "metadata": [
+                { "type" : "source" },
+                { "type" : "numberOfConnections"}
+            ] 
+        },
+        {
+            "name": "auth_connectToAWS",
+            "description": "Called when user clicks on getting ConnectToAWS from any source. e.g. explorer, developer tools, etc",
+            "metadata": [
+                { "type": "source" },
+                { "type": "numberOfConnections"}
+            ]
+        },
+        {
+            "name": "auth_awsResources",
+            "description": "Called upon user interacting with View, Modify, and Deploy AWS Resources Tab",
```

Comment:
Is "interacting" intentionally vague here or do we mean "click"?

I know in your document you clarify things, but that's not captured here. The knowledge is less likely to be "lost" when placed within the source code.

Reply:
it is interacting to capture state of user's interaction with context. It couldn't be captured by just click